### PR TITLE
fix: remove usd if NaN or 0

### PR DESCRIPTION
**Pull request by codenamejason**
Screenshot 2023-04-17 at 10 52 08 AM

### CODE REVIEW

**Review by thelostone-mc — APPROVED** [review]

**Review by nfrgosselin — CHANGES_REQUESTED** [review]
Now I'm not seeing any price data. Is that correct?

image

File: packages/round-manager/src/features/round/FundContract.tsx [hunk 1]
```diff
@@ -349,9 +353,11 @@ export default function FundContract(props: {
           <p className="text-sm">
             {" "}
             {totalAmountLeftToFund} {matchingFundPayoutToken?.name}{" "}
-            <span className="text-sm text-slate-400 ml-2">
-              ${amountLeftToFundInUSD} USD
-            </span>
+            {amountLeftToFundInUSD > 0 ? (
```
Comment (DanieleSalatti) [inline packages/round-manager/src/features/round/FundContract.tsx:356]: You should be able to use `&&` here and in other places in this PR I think?

**Review by DanieleSalatti — APPROVED** [review]
LGTM, left a minor comment

File: packages/round-manager/src/features/round/FundContract.tsx (hunk 1 above)
Comment (codenamejason) [reply packages/round-manager/src/features/round/FundContract.tsx:356]: I didn't want to show zero value

**Comment by thelostone-mc** [comment]
Just tested it locally and it worked 🙌🏼

**Pull request merged**